addons/hashes/fnc_encodeJSON.sqf

```diff
@@ -48,15 +49,16 @@ switch (typeName _object) do {
         {
             _object = [_object, _x#0, _x#1] call CBA_fnc_replace;
         } forEach [
-            ["""", "\"""],
             ["\", "\\"],
+            ["""", "\"""],
             [toString [8], "\b"],
             [toString [12], "\f"],
             [endl, "\n"],
+            [toString [10], "\n"],
             [toString [13], "\r"],
             [toString [9], "\t"]
         ];
-        str _object
+        """" + _object + """"
```

Comment:
Uh, how is this different now? I don't get it.

Reply:
`str` escapes quote signs, which we already do with backslashes.

![grafik](https://user-images.githubusercontent.com/9693842/71698842-5537a380-2dbd-11ea-9ee6-128a611efc3f.png)

Reply:
Wow, that's fucked.

Reply:
Warrants a comment.
```suggestion
        // stringify without escaping inter string quote marks
        """" + _object + """"
```